data/yam/agama/auto/sles_default_ssh_pub_key_resize_ppc64le.jsonnet

```diff
@@ -0,0 +1,60 @@
+{
```

Comment:
default is to use an 'ssh_pub_key', so could you please delete this string for existing test names and rename both create in this PR?